documentation/design-docs/dotnet-tools.md

```diff
@@ -319,15 +319,14 @@ CONVERT
         The path where the converted file is written. If unspecified the file is written in the current directory
         using the same base filename as the input file and the extension appropriate for the new format.
 
-    --to-speedscope
-        Indicates that the output file format should be speedscope JSON format used by https://www.speedscope.app/
-        Currently this is the only supported format, but other options may be available in the future.
+    -f, --format
+        Specifies the format to convert the netperf file to. Valid inputs are: speedscope.
```

Comment:
Maybe this should be:
```
Specifies the format to convert the netperf file to. Valid inputs are: netperf or speedscope.
```

Reply:
The issue being that right now we don't have a mechanism for going from X to `netperf`.  I actually throw an exception if you try to convert to `netperf`.

Reply:
The maybe fix grammar:
`Valid inputs **is** __speedscope__.`?